saw-core/src/SAWCore/Name.hs

```diff
@@ -272,26 +274,30 @@ instance Hashable Name where
 -- 'VarIndex'.
 data ExtCns e =
```

Comment:
GitHub won't let me highlight it, but the Haddocks for `ExtCns` still talk about its `VarIndex` and `NameInfo` fields, which no longer exist after this patch.

Reply:
Fixed in fc909970c2ddd64b7ed6a7c164881855fd09a1c1.